provider/gce/google/conn_network.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/juju/errors"
 	"google.golang.org/api/compute/v1"
 
-	"github.com/juju/juju/network"
+	corefirewall "github.com/juju/juju/core/network/firewall"
```

Comment:
Of course the internal package type for a gce ingress rule would be called `firewall`...